packages/dart_frog_cli/lib/src/dev_server_runner/dev_server_runner.dart

```diff
@@ -258,12 +258,17 @@ class DevServerRunner {
       );
 
       logger.detail(
-        '''[process] dart $enableVmServiceFlag $serverDartFilePath''',
+        '''[process] dart $enableVmServiceFlag $serverDartFilePath ${arguments.join(' ')}''',
       );
 
       final process = _serverProcess = await _startProcess(
         'dart',
-        [enableVmServiceFlag, '--enable-asserts', serverDartFilePath],
+        [
+          enableVmServiceFlag,
+          '--enable-asserts',
+          ...arguments,
+          serverDartFilePath,
```

Comment:
Question is there any reason why these arguments are not being pass as trailing arguments? Am I missing something?

```suggestion
          serverDartFilePath,
          ...arguments,
```

Reply:
I had a quick try:

```dart
dart my_dart_file.path --invalid-command # Runs, but with no warning about the flag
dart --invalid-command my_dart_file.path # Setting VM flags failed: Unrecognized flags: invalid_command
```

I'm happy to keep as is, wondering if this would ever introduce funky behaviour where the injected `...arguments` adds a flag that consumes the `serverDartFilePath`.